components/social_links.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { PropTypes } from 'react';
+import _ from 'lodash';
+
+import twitterIcon from '../images/icons/twitter.svg';
+import githubIcon from '../images/icons/github.svg';
+import dribbbleIcon from '../images/icons/dribbble.svg';
+
+const Icons = {
+  twitter: twitterIcon,
+  github: githubIcon,
+  dribbble: dribbbleIcon,
+};
+
+export default class SocialLinks extends React.Component {
```

Comment:
Why `React.Component`? Wouldn't it be better to change the first line to deconstruct the `Component` import?